pkg/trace/api/api.go

```diff
@@ -87,6 +90,11 @@ func NewHTTPReceiver(conf *config.AgentConfig, dynConf *sampler.DynamicConfig, o
 	if features.Has("429") {
 		rateLimiterResponse = http.StatusTooManyRequests
 	}
+	// Instantiate the AppSec HTTP handler
```

Comment:
I think the code is self-explanatory enough to not warrant this comment. Can we remove it please?

Reply:
Removed 👍